crates/rethnet_eth/src/remote/methods/hardhat.rs

```diff
@@ -11,6 +9,9 @@ use crate::{
 /// Compiler input and output structures used as parameters to Hardhat RPC methods
 pub mod compiler_io;
 
+/// input types for use with hardhat_reset
+pub mod reset;
```

Comment:
As long as the `mod` is not public, you don't need to add the docs.

Does it need to be `pub`?

Reply:
it needed to be public back when it lived in `rethnet_eth`, but now that it's co-located in `rethnet_rpc_server` it doesn't. i removed the "allow missing docs" directive in https://github.com/NomicFoundation/hardhat/pull/3998/commits/c86e0d6425c249174b9ddfa0630dff480663b03b